api/client/src/pcluster_client/model/describe_image_response_content.py

```diff
@@ -85,17 +87,20 @@ def openapi_types():
         """
         lazy_import()
         return {
-            'image_configuration': (ImageConfigurationStructure,),  # noqa: E501
             'image_id': (str,),  # noqa: E501
+            'version': (str,),  # noqa: E501
+            'image_configuration': (ImageConfigurationStructure,),  # noqa: E501
             'image_build_status': (ImageBuildStatus,),  # noqa: E501
             'region': (str,),  # noqa: E501
-            'version': (str,),  # noqa: E501
             'cloudformation_stack_status_reason': (str,),  # noqa: E501
             'imagebuilder_image_status_reason': (str,),  # noqa: E501
+            'cloudformation_stack_tags': ([Tag],),  # noqa: E501
             'imagebuilder_image_status': (ImageBuilderImageStatus,),  # noqa: E501
             'creation_time': (datetime,),  # noqa: E501
+            'cloudformation_stack_logs_arn': (str,),  # noqa: E501
```

Comment:
I am not sure we should call the parameter it cloudformation_stack_logs_arn, because it is imagebuilder build log rather than cloudformation stack log. So do you think it is better to call it like  `image_build_logs_arn`